attestation/jsonattestor/json.go

```diff
@@ -0,0 +1,213 @@
+package jsonattestor
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/cyberphone/json-canonicalization/go/src/webpki.org/jsoncanonicalizer"
+	"github.com/ghodss/yaml"
```

Comment:
This module also seems like it isn't maintained anymore. Kubernetes project has forked it [here](https://github.com/kubernetes-sigs/yaml) so I reckon let's use that 😄

Reply:
I will vendor it - I don't think it ever need to change